lib/twitter_web/router.ex

```diff
@@ -17,6 +18,14 @@ defmodule TwitterWeb.Router do
     pipe_through :browser
 
     get "/", PageController, :index
+    resources "/users", UserController, only: [:index, :show, :new, :create]
+    resources "/sessions", SessionController, only: [:new, :create, :delete]
+    resources "/posts", PostController
+
+    get "/browse/:author_id", BrowseController, :index
+
+    resources "/like/:post_id", LikeController, only: [:index, :create]
+    delete "/like/:post_id", LikeController, :delete
```

Comment:
just add delete action in resources to combine.

Reply:
will sort this later

Reply:
```suggestion
    resources "/like/:post_id", LikeController, only: [:index, :create, :delete]
```

Reply:
If I do this, like_path :delete route becomes  DELETE /like/:post_id/:id          
with an extra /:id in the end. 

My delete function searches for the like id based on the post_id, so I don't need to pass the like id. Should I modify the delete function or is there a different way to  adjust the route parameters?

Reply:
done